src/nodes/dissolve.cpp

```diff
@@ -20,3 +21,43 @@ std::string_view DissolveGraph::summary() const { return "Parent node of all sim
 
 // Return dissolve
 Dissolve &DissolveGraph::dissolve() const { return dissolve_; }
+
+// Return pair potential store
+const DoubleKeyedMap<PairPotential*> &DissolveGraph::pairPotentialStore() { return pairPotentialStore_; }
+
+/*
+ * Functions
+ */
+
+// Return maximum distance for tabulated PairPotentials
+const double DissolveGraph::pairPotentialRange() const { return pairPotentialRange_; }
+
+// Return energy kernel containing potential map
+std::unique_ptr<EnergyKernel> DissolveGraph::prepareEnergyCalculation(Configuration *cfg, std::optional<double> energyCutoff)
+{
+	// Update atom type indexing
```

Comment:
Since we'll use it a few times, and it is calculated every time, it makes sense to grab this once and re-use it.
```suggestion
    auto atomTypes = cfg->atomTypeVector();

	// Update atom type indexing
```